### PR TITLE
Status urls from Batch/Geoserver GoGoDuck in popup

### PR DESCRIPTION
**Pull request by pmbohm**
Required for the new AWS batch GoGoDuck. Will break status urls returned from Geoserver GoGoDuck

### CODE REVIEW

**Comment by pmbohm** [comment]
A cleanout is needed here starting with goggoduckV1 and therefore AsyncDownloadService.groovy, This is why this pull request starts to ignore AsyncDownloadService.groovy but many more changes are required to clean it all out.

**Comment by bhasin85** [comment]
@pmbohm How about we update the PR to make it work with both the gogoduck's? That way we can get the PR merged.

**Comment by pmbohm** [comment]
This pull requests handles `OGC:WPS--gogoduck` protocol WPS requests, with urls containing `https://wps.aodn.org.au` 
Standard Geoserver WPS GoGoDuck download requests, still handle sending emails.

**Comment by jonescc** [comment]
Perhaps looking for geoserver in the url would be more indicative of whether we are using geoserver or AWS batch (all our geoserver instances include the geoserver context).   looking for the url above will only work with the prod wps endpoint which will make testing difficult.

**Comment by jonescc** [comment]
I also wouldn't check the protocol or otherwise at some point we will have trouble deploying aws-wps netcdfoutput to test environments

**Comment by pmbohm** [comment]
@jonescc Ok so now the new workflow for AWS batch will be used when the `jobType` is 'GoGoDuck' not 'NetcdfOutput', and the server does not contain 'geoserver'

**Comment by jonescc** [comment]
I there some reason to check the protocol?   Otherwise we'll have to remember to change this when we come to migrate the netcdfoutput process.

**Comment by pmbohm** [comment]
But we haven't migrated the netcdfoutput process! So we cant use the new code for it

File: grails-app/services/au/org/emii/portal/WpsAwsService.groovy [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    }
+
+    def onResponseSuccess = { resp, xmlReader ->
+        return new StreamingMarkupBuilder().bindNode(xmlReader).toString()
+    }
+
+    def _getExecutionStatusUrl(registerResponse) {
+        def xmlLink =  new XmlSlurper().parseText(registerResponse).@statusLocation[0].toString()
+
+        return xmlLink.replace(/format=xml/,"format=html")
```
Comment (jonescc) [inline grails-app/services/au/org/emii/portal/WpsAwsService.groovy]: Returns a URL ending in format=XML not format=xml - so this doesn't work

**Comment by jonescc** [comment]
Looks good - works for both old and new wps endpoints - just one issue to fixup.

**Pull request merged**